bodhi-skopeo-lite/Dockerfile.rhel7

```diff
@@ -0,0 +1,9 @@
+# there is no Bodhi for RHEL, use Fedora instead
```

Comment:
Can we just use a single Dockerfile then?

Reply:
I wanted to reuse osbs-box base image for fedora at least to save resources. But I can drop it, it doesn't save too much.

Reply:
Changed to `fedora:29`